src/Validation/URLValidationCron.php

```diff
@@ -126,22 +102,6 @@ protected function get_event_name() {
 	 * @return string An existing interval name.
 	 */
 	protected function get_interval() {
-		return self::DEFAULT_INTERVAL_DAILY;
-	}
-
-	/**
-	 * Provides the length of time, in seconds, to sleep between validating URLs.
-	 *
-	 * @return int
-	 */
-	private function get_sleep_time() {
-
-		/**
-		 * Filters the length of time to sleep between validating URLs.
-		 *
-		 * @since 2.1
-		 * @param int The number of seconds. Default 1. Setting to 0 or a negative numbers disables all throttling.
-		 */
-		return max( (int) apply_filters( 'amp_url_validation_sleep_time', self::DEFAULT_SLEEP_TIME ), 0 );
+		return self::DEFAULT_INTERVAL_EVERY_TEN_MINUTES;
```

Comment:
I think this could be changed to run hourly, to reuse existing schedule and since it doesn't seem necessary to run quickly.